app/modules/sightings/schemas.py

```diff
@@ -59,17 +60,16 @@ class ElasticsearchSightingSchema(BaseSightingSchema):
 
     time = base_fields.Function(lambda s: s.get_time_isoformat_in_timezone())
     timeSpecificity = base_fields.Function(lambda s: s.get_time_specificity())
-    verbatimLocality = base_fields.Function(lambda s: s.get_locality())
-    locationId_id = base_fields.Function(lambda s: s.get_location_id())
+    verbatimLocality = base_fields.String(attribute='verbatim_locality')
+    locationId = base_fields.UUID(attribute='location_guid')
     locationId_value = base_fields.Function(lambda s: s.get_location_id_value())
     locationId_keyword = base_fields.Function(lambda s: s.get_location_id_keyword())
```

Comment:
Note: this function is important for Elasticsearch to work properly when you want to sort Sighting query results by location ID

Reply:
It is currently there although I'm not convinced it's working properly yet on this branch or main.....